fluentd.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 2.4'
 
+  gem.add_runtime_dependency("bundler", [">= 1.17.3", "< 2.2.0"])
```

Comment:
Why specify exact number for version constraint?
regression or critical bugs?

Reply:
> regression or critical bugs?

This is because that I'm not confirmed with more older version.
The regression nor critical bugs has nothing to do with it.

Reply:
I've removed the explicit version constraint. If regression or critical bugs were found, it should be fixed.